dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentsExportControllerTest.java

```diff
@@ -271,6 +271,16 @@ void getEnrollmentByIdNotFound()
                 .getMessage() );
     }
 
+    @Test
+    void shouldReturnABadRequestErrorWhenInvalidUidIsPassedInThePath()
+    {
+        assertEquals(
+            "Value invalidUid is not valid for parameter uid. UID must be an alphanumeric string of 11 characters",
```

Comment:
```suggestion
            "Value 'invalidUid' is not a valid UID. UID must be an alphanumeric string of 11 characters.",
```

I would quote the invalid value just in case it can contain whitespace. In that case its easier to spot.

I would also just test this once in a test like the EnumBindingTest. We can then rely on the mechanism in all controllers.